app/web/features/dashboard/CommunityBrowser.tsx

```diff
@@ -153,61 +171,56 @@ function BrowserColumn({
   selected?: number;
 }) {
   const { t } = useTranslation([DASHBOARD]);
-  const classes = useStyles();
 
   return (
-    <List className={classes.list}>
+    <StyledList>
       {parent && (
         <>
-          <ListItem
-            component={StyledLink}
-            href={routeToCommunity(parent.communityId, parent.slug)}
-          >
-            {parent.name}
-          </ListItem>
-          <Divider />
+          <StyledListItem>
+            <StyledLink
+              href={routeToCommunity(parent.communityId, parent.slug)}
+            >
+              <StyledListItemText
+                primary={parent.name}
+                sx={{
+                  color: (theme) => theme.palette.primary.main,
+                }}
```

Comment:
```suggestion
                sx={{
                  color:  theme.palette.primary.main,
                }}
```

and add this import up top

`import { theme } from "theme";
`